src/main/java/uk/gov/ida/rp/testrp/filters/SecurityHeadersFilter.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.ida.rp.testrp.filters;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerResponseContext;
+import javax.ws.rs.container.ContainerResponseFilter;
+import java.io.IOException;
+
+public class SecurityHeadersFilter implements ContainerResponseFilter {
```

Comment:
Given that this is the same as for stub-idp, is there any common place we can put this and reference it from both?